layouts/auto_generated_product.tpl

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+{% include "template-variables" layout_auto_rendered_product: true %}
+{% include "product-variables" %}
+
+<html class="layout-root {{ language_flags_mode }} {{ language_names_mode }} {{ language_menu_mode }} {{ site_search_mode }}" lang="{{ page.language_code }}" data-view-state="{{ view_mode }}" data-lingual-mode="{{ site_lingual_mode }}" data-menu-main-state="closed">
+  <head prefix="og: http://ogp.me/ns#">
+    {% include "template-head" layout_auto_rendered_product: true %}
+  </head>
+
+  {% capture bottom_content_html %}
+    {% unless editmode %}
+      {% content bind=product name="content" %}
+    {% endunless %}
+  {% endcapture %}
+
+  {% capture bottom_content_size %}
+    {{ bottom_content_html | size | minus: 1 }}
+  {% endcapture %}
+
+  {% unless bottom_content_size contains "-" %}
+    {% assign bottom_content_has_content = true %}
+  {% endunless %}
```

Comment:
This is always `true`, because the `capture` block also takes into account the whitespace the tags generate, hence the size will not be 0.

This can also be simplified by writing it like this:
```liquid
{% capture bottom_content_html %}
  {% unless editmode %}
    {% content bind=product name="content" %}
  {% endunless %}
{% endcapture %}

{% assign bottom_content_size = bottom_content_html | strip | size %}
```
and whenever you need to check if the bottom_content has content, you just do `if bottom_content_size > 0`.